code/modules/research/designs/medical_designs.dm

```diff
@@ -292,7 +292,7 @@
 
 /datum/design/cyberimp_breather
 	name = "Breathing Tube Implant"
-	desc = "This simple implant adds an internals connector to your back, allowing you to use internals without a mask and protecting you from being choked."
+	desc = "This implant allows the user to connect to internals without wearing a mask, while also protecting from being choked."
```

Comment:
I'm not sure why this description was changed?

Reply:
It was changed to be more in line with the other implants that just say what they do without basically a lore justification on HOW it does that.